### ISSUE TITLE
Document representaion.

### ISSUE THREAD

**Issue by sekhar14** [cemoody/lda2vec#57]
Giving same vector representation for every document. Has anyone faced the same problem?

**Comment by kkrasnas** [comment]
Yes! After some epochs, the dirichlet loss freezes and every document belongs to all topics equally.

**Comment by nateraw** [comment]
Has anybody been able to fix this? I rewrote this in tensorflow but it seems the logic or process is flawed somewhere.

**Comment by duolinwang** [comment]
Yes！ I got the same problem!!! What is the solution????
